src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +27,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = Person.nextSequenceNumber;
+        Person.nextSequenceNumber++;
```

Comment:
Hi, I did exactly that when creating a new branch. But these changes from another branch still showed up on the new branch